core/trino-spi/src/main/java/io/trino/spi/StandardErrorCode.java

```diff
@@ -196,6 +196,7 @@ public enum StandardErrorCode
     ADMINISTRATIVELY_PREEMPTED(131080, INSUFFICIENT_RESOURCES),
     EXCEEDED_SCAN_LIMIT(131081, INSUFFICIENT_RESOURCES),
     EXCEEDED_TASK_DESCRIPTOR_STORAGE_CAPACITY(131082, INSUFFICIENT_RESOURCES),
+    EXCEEDED_CONNECTOR_MEMORY_LIMIT(131083, INSUFFICIENT_RESOURCES),
```

Comment:
"exceeded connector memory limit" seems wrong. There's no per-connector memory limit. If I'm understanding the rest of the code correctly, this would actually be "connector memory reservation exceeded node memory limit".